eng/pipeline/cleanup-acr-images.yml

```diff
@@ -0,0 +1,75 @@
+# This pipeline is used to clean up old images in our Azure Container Registries.
+# It's derived from https://github.com/dotnet/docker-tools/blob/main/eng/pipelines/cleanup-acr-images.yml
+
+trigger: none
+pr: none
+
+schedules:
+- cron: "0 5 * * *"
+  displayName: Nightly build
+  branches:
+    include:
+    - microsoft/main
```

Comment:
Fix formatting: missing indentations before list elements.

Reply:
Same issue throughout the file.

Reply:
I've run prettier now, it looks much better